charts/nr-k8s-otel-collector/templates/daemonset-configmap.yaml

```diff
@@ -0,0 +1,309 @@
+                enabled: false
+              system.disk.weighted_io_time:
+                enabled: false
+          network:
+            metrics:
+              system.network.connections:
+                enabled: false
+          processes:
+          process:
+            metrics:
+              process.cpu.utilization:
+                enabled: true
+              process.cpu.time:
+                enabled: false
+            mute_process_name_error: true
+            mute_process_exe_error: true
+            mute_process_io_error: true
+            mute_process_user_error: true
+
+      kubeletstats:
+        collection_interval: 20s
+        auth_type: "serviceAccount"
+        endpoint: "${KUBE_NODE_NAME}:10250"
+        insecure_skip_verify: false
+        metrics: 
+          k8s.container.cpu_limit_utilization:
+            enabled: true 
+      prometheus:
+        config:
+          scrape_configs:
+            - job_name: cadvisor
+              bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+              kubernetes_sd_configs:
+                - role: node
+              relabel_configs:
+                - replacement: kubernetes.default.svc.cluster.local:443
+                  target_label: __address__
+                - regex: (.+)
+                  replacement: /api/v1/nodes/$${1}/proxy/metrics/cadvisor
+                  source_labels:
+                    - __meta_kubernetes_node_name
+                  target_label: __metrics_path__
+              scheme: https
+              tls_config:
+                ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+                insecure_skip_verify: false
+                server_name: kubernetes
+            - job_name: kubelet
+              bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+              kubernetes_sd_configs:
+                - role: node
+              relabel_configs:
+                - replacement: kubernetes.default.svc.cluster.local:443
+                  target_label: __address__
+                - regex: (.+)
+                  replacement: /api/v1/nodes/$${1}/proxy/metrics
+                  source_labels:
+                    - __meta_kubernetes_node_name
+                  target_label: __metrics_path__
+              scheme: https
+              tls_config:
+                ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+                insecure_skip_verify: false
+                server_name: kubernetes
+      filelog:
+        include:
+          - /var/log/pods/*/*/*.log
+          - /var/log/*/*
+          - /var/log/*
+          - /var/log/alternatives.log
+          - /var/log/cloud-init.log
+          - /var/log/auth.log
+          - /var/log/dpkg.log
+          - /var/log/syslog
+          - /var/log/messages
+          - /var/log/secure
+          - /var/log/yum.log
+        exclude:
+          # Exclude logs from opentelemetry containers
```

Comment:
why?
how a customer could check the logs from the monitoring?

If it is to avoid "creating a loop" I think the way to go is not exporting metrics to logs.

Reply:
actually I have some doubts about this 🤔 Not an easy choice!

Reply:
The choice is easy. You should not duplicate or use the same thing for input and output. As it is a daemonset and it has to mount a folder with the logs files to be sent to the NR. You want to send the data of the collector to NR too.

 - The pod should not write to a file, all the logging should to to stdout.
 - The collector should send every pod in the cluster. Itself included.
 - Do not log the logging pipeline so there is no loop. (It will also duplicate ALL the logs)
 - In case debug logs are needed, create a new log pipeline that includes all the debug information without loop possibility.

Reply:
honestly, i assumed the customer would check the logs from the monitoring by accessing the logs in the pods themselves but perhaps this is not the way to think about it? I'm new to thinking about how customers would interact with the agents, but given that our customers are engineers... I would think that accessing logs in a pod would be basic knowledge, and given that customers would have to pay for that data to be written into new relic, i assumed they would prefer to not have the otel pod logs sent to NR. 

we can add a separate debug logging pipeline in a later PR, if it is deemed necessary to do, i don't think that should be a blocker on this PR, especially since the loop possibility is removed.

Reply:
What is our actual proposal to scrape monitoring logs from nri_bundle?
Shouldn't we do the same?

Reply:
That should be asked to the logging team as they are the ones that understand how fluentbit works.

We never had to go this far. This is why this is becoming tricky.